bin/spark-submit

```diff
@@ -44,7 +44,10 @@ while (($#)); do
   shift
 done
 
-DEFAULT_PROPERTIES_FILE="$SPARK_HOME/conf/spark-defaults.conf"
+if [ ! -d "$SPARK_CONF_DIR" ]; then
```

Comment:
actually, maybe this should be "if not defined" instead of "if not exists". Otherwise if the user sets `SPARK_CONF_DIR` to a directory that doesn't exist then this will silently use a different config.